lib/rules/function-url-no-scheme-relative/__tests__/index.js

```diff
@@ -69,6 +69,10 @@ testRule(rule, {
     {
       code: "a { background: url(var(--image)); }",
       description: "ignore variable"
+    },
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.